ironic.conf.j2

```diff
@@ -21,6 +21,8 @@ use_stderr = true
 require_agent_token = true
 # NOTE(dtantsur): the default md5 is not compatible with FIPS mode
 hash_ring_algorithm = sha256
+my_ip = {{ env["IRONIC_IP"] }}
```

Comment:
very nit: you can use dot access in jinja (`env.IRONIC_IP`).

Reply:
I changed it, thanks